README.md

```diff
@@ -123,6 +123,50 @@ For example, you can add a step before running the Changesets GitHub Action:
     NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 ```
 
+#### Custom Publishing
+
+If you want to hook into when publishing should occur but have your own publishing functionality you can 
+utilize the `hasChangesets` output.
```

Comment:
```suggestion
If you want to hook into when publishing should occur but have your own publishing functionality you can utilize the `hasChangesets` output.

Not that you might need to account for things already being published in your script because a commit without any new changesets can always land on your base branch after a successful publish. In such a case you need to figure out on your own how to skip over the actual publishing logic or handle errors gracefully as most package registries won't allow you to publish over already published version.
```